src/sympc/tensor/mpc_tensor.py

```diff
@@ -736,6 +756,34 @@ def _get_shape(
         res = op(x, y, **kwargs_)
         return res.shape
 
+    def truncation(
+        self, result: "MPCTensor", op_str: str, is_private: bool
```

Comment:
MAYBE: This should be changed to another name and assign `result` inside this function